setup.py

```diff
@@ -16,7 +16,7 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    'html5lib>=0.99999999,>=1.0b9',
```

Comment:
I'm not sure why it works, but it does.

If I do an install, pip will pull 1.0b10. If I install html5lib== the 8-nines version, pip check will say it's alright. If I install b8 or a 7-nines version pip check complains of a broken dependency.